tests/multiparser_js_html/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -513,5 +584,38 @@ const closingScriptTag2 = /* HTML */ \`
   </script>
 \`;
 
+html\`
+  <div
+    style="
+ \${foo}
+"
+  ></div>
+\`;
+html\`
+  <div style=\${foo}></div>
+\`;
+
+html\`
+  <div
+    style="   color : red;
+            display    :inline "
+  ></div>
+\`;
+
+html\`
+  <div
+    style="   color : red;
+\${foo}
+            display    :inline "
+  ></div>
+\`;
+html\`
+  <div
+    style="   color : red;
+\${foo}:\${bar};
+            display    :inline "
```

Comment:
Why it is ugly?

Reply:
I disabled format on multi-parser for now, it could be `<div style="${foo}">` -> `<div style="HTML_PLACEHOLDER">` may leads to a parser error.

Let's work on html format and ignore the complicated case first.

Reply:
Okay, we should keep issues are open to future improvements